flashlight/lib/sequence/CMakeLists.txt

```diff
@@ -2,9 +2,68 @@ cmake_minimum_required(VERSION 3.10)
 
 # ------------------------- Components -------------------------
 
+add_library(
+  fl_lib_sequence
+  ""
+  )
+
+target_include_directories(
+  fl_lib_sequence
+  PRIVATE
+  ${PROJECT_SOURCE_DIR}
+  )
+
+# Only build the library if the FL CUDA backend is enabled.
+if (NOT FL_USE_CUDA)
+  message(STATUS "FL_BACKEND is not CUDA - forcing FL_LIBRARIES_USE_CUDA to OFF")
+  set(FL_LIBRARIES_USE_CUDA OFF)
+endif()
+
 # criterion-library
 include(${CMAKE_CURRENT_LIST_DIR}/criterion/CMakeLists.txt)
 
 set(FL_LIBRARIES_SEQUENCE_CUDA_SOURCES
   ${FL_LIBRARIES_SEQUENCE_CRITERION_CUDA_SOURCES}
   )
+set(FL_LIBRARY_CUDA_SOURCES
```

Comment:
I think one of `FL_LIBRARIES_SEQUENCE_CUDA_SOURCES` or `FL_LIBRARIES_SEQUENCE_CUDA_SOURCES` redundant or empty having moved this stuff and can be removed